README.md

```diff
@@ -1 +1,57 @@
-# React-Redux Space Travelers-Hub
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# SPACE TRAVELERS' HUB
+
+> This educational project is to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+
+## Built With
+
+- ReactJS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- `git@github.com:myaserkhan/React-Redux-Space-Travelers-Hub.git`
+- `cd React-Redux-Space-Travelers-Hub`
+- `npm install`
+- `npm start`
+
+## Prerequisites
+
+- Node.js
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@myaserkhan](https://github.com/githubhandle)
+- Twitter: [@khanyaser007](https://twitter.com/twitterhandle)
+- LinkedIn: [myaserkhan](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@johnkioko](https://github.com/githubhandle)
+- Twitter: [@johnmutisya](https://twitter.com/twitterhandle)
+- LinkedIn: [John Mutisya](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Great job team 🤝

- [ ] [Optional] You may also fix the MIT license link as it appears broken at the moment. This will make your readme file very professional